gamemodes/terrortown/gamemode/shared/lang/en.lua

```diff
@@ -1276,7 +1276,7 @@ L.dna_help_primary = "Collect a DNA sample"
 L.dna_help_secondary = "Switch the DNA slot"
 L.dna_help_reload = "Delete a sample"
 
-L.binoc_help_pri = "Identify a body."
+L.binoc_help_pri = "Search a body."
```

Comment:
```suggestion
L.binoc_help_pri = "Inspect a body."
```
Regarding line 1266 this seems like the most consistent wording

Reply:
Regarding string 1040 (and many others), there should be the word «search». While the word «inspect» is used only in one string (1266 and currently 1279).

Reply:
you are right, in that case line 1266 should be `search` i guess. unless i'm missing something regarding the meaning behind these two

Reply:
Is it the same action as in line 1267, right? If yes, then I will change it to "search".

Reply:
Afaik search and inspect are used interchangeably. Just make sure search/inspect are not mixed with confirm.